english/hawkstrider/startr/startr.md

```diff
@@ -0,0 +1,11 @@
+The hawkstrider pauses for a moment then says:
+
+"Reaarrr. Urrrrr."
+
+A. [How ya doin fam?](../error/error.md)
+
+B. [Doral ana-diel?](../tier1/tier2/tier2.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.